In [29]:
import seaborn as sns
titanic = sns.load_dataset("titanic")   
import pandas as pd

print(titanic.shape)


(891, 15)


In [30]:
print(titanic.dtypes)


survived          int64
pclass            int64
sex                 str
age             float64
sibsp             int64
parch             int64
fare            float64
embarked            str
class          category
who                 str
adult_male         bool
deck           category
embark_town         str
alive               str
alone              bool
dtype: object


In [31]:
print(titanic.isna().sum()[titanic.isna().sum() > 0])


age            177
embarked         2
deck           688
embark_town      2
dtype: int64


In [32]:
print(titanic["class"].value_counts())

class
Third     491
First     216
Second    184
Name: count, dtype: int64


les 3 constats que j'ai fait:
-la colonne deck contient le plus grand nombre de manquants
-on a plus de passager en 3ème colonne que dans les autres colonnes
-on compte au total 891 passagers repartis dans 15 colonnes.